tests/analysis.spec.ts

```diff
@@ -87,29 +88,30 @@ describe('Functional test of analysis', () => {
         expect(bundle.analysisResults.timing.analysis).toBeGreaterThanOrEqual(0);
         expect(bundle.analysisResults.timing.fetchingCode).toBeGreaterThanOrEqual(0);
         expect(bundle.analysisResults.timing.queue).toBeGreaterThanOrEqual(0);
-        expect(new Set(bundle.analysisResults.coverage)).toEqual(
-          new Set([
-            {
-              files: 2,
-              isSupported: true,
-              lang: 'Java',
-            },
-            {
-              files: 1,
-              isSupported: true,
-              lang: 'C++ (beta)',
-            },
-            {
-              files: 6,
-              isSupported: true,
-              lang: 'JavaScript',
-            },
-          ]),
-        );
+        expect(bundle.analysisResults.coverage).toIncludeSameMembers([
+          {
+            files: 2,
+            isSupported: true,
+            lang: 'Java',
+            type: 'SUPPORTED',
```

Comment:
Does coverage send `type` after the change now?

Reply:
No, it's not sending anything new, those fields were just ignored in the previous comparisons 🙈